src/ipc/ipc4/helper.c

```diff
@@ -911,11 +911,11 @@ const struct comp_driver *ipc4_get_drv(const uint8_t *uuid)
 		}
 	}
 
-	tr_err(&comp_tr, "get_drv(): the provided UUID (%08x %08x %08x %08x) can't be found!",
-	       *(uint32_t *)(&uuid[0]),
-	       *(uint32_t *)(&uuid[4]),
-	       *(uint32_t *)(&uuid[8]),
-	       *(uint32_t *)(&uuid[12]));
+	tr_warn(&comp_tr, "get_drv(): the provided UUID (%08x %08x %08x %08x) can't be found!",
```

Comment:
> When ipc4_get_drv() fails to find a driver, it might mean, that the
> driver needs to be linked dynamically.

But why it is not linked already in that case?

Reply:
because we link them *dynamically* - at *run-time*. Please check logs and source - this error / warning comes before the library manager attempts to load a missing driver. If it succeeds - all good, we proceed. So, this isn't a failure yet.